src/main/java/org/owasp/html/HtmlEntities.java

```diff
@@ -2349,6 +2454,13 @@ public static int appendDecodedEntity(
     }
   }
 
+  private static boolean isHtmlIdContinueChar(char ch) {
```

Comment:
@mikesamuel 
What is the role of this method?

for example in this below case,
```java
assertEquals(
            "&#\n",
            Encoding.decodeHtml("&#&#xa;"));
```
I don't know how it depends on the result of isHtmlIdContinueChar.

```java
if (!isHtmlIdContinueChar(digit)) {
    if (i == offset + 2) {
        codepoint = -1;
    }
    if (digit == ';') {
        i += 1;
    }
    tail = i;
    break;
}
```

Reply:
I find it. in below case, `&#x6a` check is needed. I got it.
```java
String attackString = "<a href=\"&#x6a;avascript:alert(1)\">XSS</a>";
String s = Encoding.decodeHtml(attackString);
```